test/main.cpp

```diff
@@ -0,0 +1,11 @@
+#include <SDL.h>

```

Comment:
Is this include needed?

Reply:
Interesting.
Before it was needed cause SDL redefined main but now it isn't.
So just removed it 🙂